src/client/pps.go

```diff
@@ -14,28 +14,62 @@ func NewJob(jobID string) *pps.Job {
 	return &pps.Job{ID: jobID}
 }
 
-type InputType int
+var (
+	MapStrategy = &pps.Strategy{
+		Partition:      pps.Partition_BLOCK,
+		Incrementality: true,
+	}
+
+	ReduceStrategy = &pps.Strategy{
+		Partition:      pps.Partition_FILE,
+		Incrementality: false,
+	}
+
+	StreamingReduceStrategy = &pps.Strategy{
+		Partition:      pps.Partition_FILE,
+		Incrementality: true,
+	}
```

Comment:
I think I was the one that pushed for `StreamingReduce` but looking at it now I think `IncrementalReduce` makes a bit more sense.

Also I think we should name the field `Incremental` rather than `Incrementality`.